### PR TITLE
more accurate sign message

### PR DESCRIPTION
**Pull request by karlthepagan**
Tested!

image

This might be "undefined" for a few ticks after first spawn, but meh.

### CODE REVIEW

File: parameter.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ let mod = {
     COST_MATRIX_VALIDITY: 1000,
     CONSTRUCTION_PRIORITY: [STRUCTURE_SPAWN,STRUCTURE_EXTENSION,STRUCTURE_LINK,STRUCTURE_STORAGE,STRUCTURE_TOWER,STRUCTURE_ROAD,STRUCTURE_CONTAINER,STRUCTURE_EXTRACTOR,STRUCTURE_WALL,STRUCTURE_RAMPART],
     CONTROLLER_SIGN: false,
-    CONTROLLER_SIGN_MESSAGE: "Territory of the Open Collaboration Society! (https://github.com/ScreepsOCS)",
+    CONTROLLER_SIGN_MESSAGE: `Territory of ${_.chain(Game.spawns).values().first().get('owner.username').value()}, an Open Collaboration Society user! (https://github.com/ScreepsOCS)`,
```
Comment (Spedwards) [inline parameter.js]: Not sure why you're using `_.chain` instead of `_` but the changes are nonetheless fine.

Comment (karlthepagan) [inline parameter.js]: unfortunately not using _.chain means that .first() returns a literal value which will not provide .get()

Comment (Spedwards) [inline parameter.js]: So use `_(Game.spawns).values().first().owner.username`?

Comment (karlthepagan) [inline parameter.js]: .get will be undef safe!

Comment (Spedwards) [inline parameter.js]: Live on the edge a little. ;)

**Pull request merged**